client/state/billing-transactions/test/actions.js

```diff
@@ -100,62 +97,4 @@ describe( 'actions', () => {
 			} );
 		} );
 	} );
-
-	describe( '#sendBillingReceiptEmail()', () => {
```

Comment:
This test doesn't need to be completely removed, it can still be useful. There's the REST request we can verify with `nock`, and actions dispatched to show notices.

Reply:
Sure thing. It felt a bit unnecessary to me, but I was to do it - I've reverted it an updated the test to check for notices in 5f711159082c3def742916df1ccdf185f1054800.